api/app/user/rest.py

```diff
@@ -92,6 +93,14 @@ def update_user_attribute(user_id):
     return jsonify(data=user_to_update.serialize()), 200
 
 
+@user_blueprint.route('/<uuid:user_id>/archive', methods=['POST'])
+def archive_user(user_id):
```

Comment:
what's the permission necessary to perform this?

Reply:
`user_blueprint.before_request(requires_admin_auth)` —so yeah checks that it comes from admin.